app/src/main/java/com/example/android/receitaboa/activity/EditarReceitaActivity.java

```diff
@@ -212,68 +210,74 @@ protected void onActivityResult(int requestCode, int resultCode, @Nullable Inten
                     fotoReceita.compress(Bitmap.CompressFormat.JPEG,75,baos);
                     byte[] dadosImagemReceita = baos.toByteArray();
 
-                    //Salvar imagem firebaseStorage
-                    StorageReference fotoReceitaRef = storageRef
-                            .child("imagens")
-                            .child("receitas")
-                            .child(identificadorChef) //idChefAuth
-                            .child(receitaId + ".jpg");
+                        //Salvar imagem firebaseStorage
+                        StorageReference fotoReceitaJPGRef = storageRef
+                                .child("imagens")
+                                .child("receitas")
+                                .child(identificadorChef) //idChefAuth
+                                .child(receitaId + ".jpg");
+
+                        UploadTask uploadTask = fotoReceitaJPGRef.putBytes(dadosImagemReceita);
 
-                    UploadTask uploadTask = fotoReceitaRef.putBytes(dadosImagemReceita);
-                    //abrirDialogCarregamento("Carregando a foto, aguarde!"); //impede o que o chef atrapalhe o processo de salvar a imagem no FirebaseStorage
+                        uploadTask.addOnFailureListener(new OnFailureListener() {
+                            @Override
+                            public void onFailure(@NonNull Exception e) {
 
-                    uploadTask.addOnFailureListener(new OnFailureListener() {
-                        @Override
-                        public void onFailure(@NonNull Exception e) {
-                            Toast.makeText(EditarReceitaActivity.this,
-                                    "Erro ao salvar a imagem, tente novamente!",
-                                    Toast.LENGTH_SHORT).show();
-                        }
-                    }).addOnSuccessListener(new OnSuccessListener<UploadTask.TaskSnapshot>() {
-                        @Override
-                        public void onSuccess(UploadTask.TaskSnapshot taskSnapshot) {
+                                dialog.dismiss();
+                                Toast.makeText(EditarReceitaActivity.this,
+                                        "Erro ao salvar a imagem, tente novamente!",
```

Comment:
Não se deve pôr uma string diretamente dentro do programa, extrair a string para um arquivo de tradução (string.xml)

https://www.andtuts.com/why-and-how-to-use-string-resources/